userdocs/src/getting-started/clean-up.md

```diff
@@ -4,21 +4,23 @@ During testing, or maybe when a project has come to an end - there might be a ne
 ## Delete a cluster
 
 ```bash
-# Delete an okctl cluster. Format:
-# okctl delete cluster <environment name>
-#
-# <cluster name>     must equal the environment name used for cluster creation.
-#
+# Usage
+okctl --cluster-declaration <path to cluster declaration> delete cluster
+
 # Example:
-okctl delete cluster test
+okctl --cluster-declaration cluster-yaml delete cluster
 ```
 
 ### Delete hosted zone
 
 If you want to remove your cluster's primary hosted zone, use
 
 ```bash
-okctl delete cluster <environment name> --i-know-what-i-am-doing-delete-hosted-zone-and-records true 
+# Usage
+okctl --cluster-declaration <path to cluster declaration> delete cluster --i-know-what-i-am-doing-delete-hosted-zone-and-records true 
+
+# Example
+okctl --cluster-declaration cluster.yaml delete cluster --i-know-what-i-am-doing-delete-hosted-zone-and-records true 
```

Comment:
Haven't we removed the `i-know-what-i-am-doing-delete-hosted-zone-and-records` some time ago? Don't we always delete the hosted zone these days?

Reply:
oh right. nice catch